### PR TITLE
Meta tweaks

### PR DESCRIPTION
**Pull request by hafffe**

### CODE REVIEW

File: license [hunk 1]
```diff
@@ -1,21 +1,9 @@
-The MIT License (MIT)
+The MIT License (MIT)MIT License
```
Comment (kevva) [inline license]: 🐈

File: test.js [hunk 2]
```diff
@@ -1,5 +1,5 @@
 import test from 'ava';
-import fn from './';
+import fn from '.';
```
Comment (kevva) [inline test.js]: `fn` => `m`

**Review by kevva — CHANGES_REQUESTED** [review]

**Comment by kevva** [comment]
Can you remove `--save` from the install script too in the docs? And also change `http` to `https` in the link under the license section.

File: license (hunk 1 above)
Comment (hafffe) [reply license]: ✔

**Comment by kevva** [comment]
Thanks @alonalon 🥇

**Pull request merged**